phusion-ruby-fips/Dockerfile

```diff
@@ -43,12 +44,17 @@ RUN apt-mark hold openssl
 RUN apt-mark hold libssl1.0.0
 RUN apt-mark hold libssl1.1
 
+# Install SASL dependency for OpenLDAP  
+RUN apt-get install -y libsasl2-dev && \
+    apt-mark hold libssl1.1
```

Comment:
Duplicate line see line number 45.

Reply:
It's not a duplicate. I'm not sure the reason, but installing `libsasl2-dev` removes the hold on `libssl1.1` so it has to be marked again after installation.